editor/plugins/material_editor_plugin.cpp

```diff
@@ -95,13 +95,17 @@ void MaterialEditor::_button_pressed(Node *p_button) {
 		sphere_instance->hide();
 		box_switch->set_pressed(true);
 		sphere_switch->set_pressed(false);
+		EditorSettings::get_singleton()->set_setting("_preview_on_sphere", false);
+		EditorSettings::get_singleton()->save();
```

Comment:
This shouldn't be saved as a setting, but as project metadata.
```suggestion
		EditorSettings::get_singleton()->set_project_metadata("inspector_options", "material_preview_on_sphere", false);
```

Reply:
changed in 60b14c2